README.md

```diff
@@ -42,6 +42,85 @@ func main() {
 	...
 ```
 
+## How to Start ##
+
+Line Messaging API use JSON to format data.
+```ParseRequest()``` will help you to parse the ```*http.Request``` content and return a slice of Pointer point to Event Object.
+
+```go
+	events, err := bot.ParseRequest(req)
+	if err != nil {
+		// Do something when something bad happened.
+	}
+```
+
+Line Messaging API define 7 types of event - ```EventTypeMessage```, ```EventTypeFollow```, ```EventTypeUnfollow```, ```EventTypeJoin```, ```EventTypeLeave```, ```EventTypePostback```, ```EventTypeBeacon```. You can check this by  ```event.Type```
+
+```go
+	for _, event := range events {
+		if event.Type == linebot.EventTypeMessage {
+			// Do Something...
+		}
+	}
+```
+
+### Receiver ###
+
+To send a message to user/group/room, you might need an ID
+
+```go
+	userID := event.Source.UserID
+	groupID := event.Source.GroupID
+	RoomID := event.Source.RoomID
+```
+
+or reply token
+
+```go
+	replyToken := event.ReplyToken
+```
+
+### Create Message ###
+
+Line Messaging API supply many types of message, just use the ```New<Type>Message()``` to create it.
```

Comment:
The LINE Messaging API provides various types of messages. To create a message use ```New<Type>Message()```.